shared/wallets/send-form/participants/container.js

```
@@ -152,7 +165,7 @@ const ParticipantsChooser = props => {
     declare var ifFlowErrorsHereItsCauseYouDidntHandleAllActionTypesAbove: (recipientType: empty) => any
     ifFlowErrorsHereItsCauseYouDidntHandleAllActionTypesAbove(props.recipientType);
     */
-      return null
+      throw new Error(`Unexpected recipientType ${props.recipientType}`)
```

better to throw an error than to silently fail